ONSdigital/rasrm-acceptance-tests
# Feature tests for respondent on response status

# Motivation and Context
Feature tests for adding respondent name to the response status page in R-ops.

# What has changed
- Added feature file `display_respondent_alongside_response_status`

# How to test?
- Run with the [Response-ops PR](https://github.com/ONSdigital/response-operations-ui/pull/267) and run the acceptance tests. All should pass
# Links
[Trello](https://trello.com/c/1rcfmtok)
